pandas/tests/io/conftest.py

```diff
@@ -37,6 +41,14 @@ def s3_resource(tips_file, jsonl_file):
     """
     pytest.importorskip('s3fs')
     boto3 = pytest.importorskip('boto3')
+    botocore = pytest.importorskip('botocore')
+    if (LooseVersion(botocore.__version__) < LooseVersion("1.11.0")
```

Comment:
just make the minimum in the tests 1.11 then u don’t need this at all

Reply:
You mean add it to the `travis-36` dependencies? That would currently fail due to #23754

Reply:
yes

Reply:
Adding `botocore>=1.11` to the dependencies will mean either failures due to #23754 (which is most likely an upstream moto-bug), or that *none* of the boto tests are actually run (because they'd be skipped). The `travis-36` build is the *only* build testing `boto`. 

With this construct (and I admit it's not pretty), we could have one build doing `botocore<1.11` (actually testing the code), and one with `botocore>=1.11`, which would be silently skipping them now but will start working again as soon as the moto bug is fixed and a new version available.

Reply:
I haven't been following the boto / moto issues closely, but it seems like this is the best option for now if we want to have any of the boto stuff actually tested.

Reply:
botocore is only a test dep, so i don't mind switching it to a higher version. Then simply add this to other builds until we are actually testing this.

Reply:
From what I understand there's a conflict between the latest boto/botocore and moto
https://github.com/pandas-dev/pandas/issues/23754

We could also try to fix this at the `PANDAS_TESTING_MODE` level. That adds a `warnings.simplefilter` at the start of the test. Perhaps the fixture could add an ignore to our filters? Though maybe @h-vetinari already tried that.